### PR TITLE
Commit to close issue #22 enhancements from user feedback

### PR DESCRIPTION
**Pull request by daveeloo**

### CODE REVIEW

File: greenhouse/templates/person.html [hunk 1]
```diff
@@ -51,6 +51,7 @@
         	| <a id="unify" title="If this person already exists under another identity
         	click here to merge the identities"> [unify duplicate identity] </a>
         </li>
+        <li><b>DDPO:</b><a href="http://qa.debian.org/developer.php?email=paultag@ubuntu.com"> {{person.name}}</a></li>
```
Comment (paulproteus) [inline greenhouse/templates/person.html]: It's weird that this hard-codes a link to paultag.

File: greenhouse/templates/person.html [hunk 2]
```diff
@@ -82,6 +83,25 @@
...
+
+<table>
+    <tr>
+        <th>Package</th>
+        <th>Version</th>
+        <th>Date</th>
+    </tr>
+    {% for ul in recent_uploads %}
+    <tr>
+        <td><a href="https://launchpad.net/ubuntu/+source/{{ ul.package }}/{{ ul.version }}">{{ ul.package }}</a></td>
```
Comment (paulproteus) [inline greenhouse/templates/person.html]: If this is for Debian, it's weird that this links to Launchpad and Ubuntu.

File: greenhouse/uploads/views.py [hunk 3]
```diff
@@ -208,31 +211,42 @@ def dashboard():
...
+    query = Q()
+    for object_pk in contacted_filter:
+        query |= Q(id=object_pk)
+    contacted = People.objects.filter(query) if query else []
     return {'first_timers': first_timers,
             'experienced': experienced,
-            'inactive': inactive}
+            'inactive': inactive,
+            'contacted': contacted,
+            'actions': actions,
```
Comment (paulproteus) [inline greenhouse/uploads/views.py:248]: So beautiful! Thanks for fixing this particular style issue.

File: greenhouse/templates/person.html (hunk 1 above)
Comment (paultag) [inline greenhouse/templates/person.html]: (Muhahaha)

File: greenhouse/templates/person.html [hunk 4]
```diff
@@ -82,6 +83,25 @@
...
+<table>
+    <tr>
+        <th>Package</th>
+        <th>Version</th>
+        <th>Date</th>
+    </tr>
+    {% for ul in recent_uploads %}
+    <tr>
+        <td><a href="http://packages.qa.debian.org/{{ ul.package }}">{{ ul.package }}</a></td>
+        <td>{{ ul.version }}</d>
```
Comment (paulproteus) [inline greenhouse/templates/person.html]: <pre>
</d> != </td>
</pre>

**Comment by paulproteus** [comment]
Yippee. Thanks for improving this!

**Pull request merged**